src/nikhilMJun20/MissingNumberFromArray.java

```diff
@@ -0,0 +1,28 @@
+/*Find the missing number in integer array of 1 to 10?
+Hint: You have given an integer array which contains numbers from 1 to 10 but one number is missing, you need to write a Java program to find that missing number in an array.
+*/
+
+package nikhilMJun20;
+
+public class MissingNumberFromArray {
+	void findMissing(int[] intArray) {
+		int originalSum=0, arraySum=0;
+		
+		System.out.println("Given Array : {1, 2, 3, 4, 6, 7, 8, 9, 10}");
+		for(int index=0; index<intArray.length; index++) {
+			arraySum += intArray[index];
+		}
+		
+		for(int num=1; num<=10; num++) {
```

Comment:
--This logic also looks good to me.
-- You can also make this work using only one for loop.

Reply:
->This was exactly for 1 to 10 hard coded. So, I did not use n(n+1)/2 formula.

Reply:
-- Sure then code looks good to me

Reply:
--okay good then this logic also works